Userland/Libraries/LibCore/CMakeLists.txt

```diff
@@ -19,29 +19,30 @@ set(SOURCES
     NetworkJob.cpp
     Notifier.cpp
     Object.cpp
-    Process.cpp
-    ProcessStatisticsReader.cpp
     Property.cpp
     SecretString.cpp
     SessionManagement.cpp
     SOCKSProxyClient.cpp
     StandardPaths.cpp
     Stream.cpp
     System.cpp
-    SystemServerTakeover.cpp
     TCPServer.cpp
     TempFile.cpp
     Timer.cpp
     UDPServer.cpp
     Version.cpp
 )
+
 if (NOT ANDROID AND NOT WIN32 AND NOT EMSCRIPTEN)
     list(APPEND SOURCES
         Account.cpp
         FilePermissionsMask.cpp
         GetPassword.cpp
         Group.cpp
         LocalServer.cpp
+        Process.cpp
+        ProcessStatisticsReader.cpp
+        SystemServerTakeover.cpp
```

Comment:
do these not compile for android and emca? if they do maybe these should just be excluded for win32?

Reply:
They compile, but they aren't particularly useful. I can pull them out though, as they are mostly an issue due to their POSIX-ness.